shared/fixes/ansible/mount_option_krb_sec_remote_filesystems.yml

```diff
@@ -0,0 +1,20 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+
+- name: "Get nfs and nfs4 mount points, that don't have Kerberos security option"
+  shell: grep "[[:space:]]nfs[4]*[[:space:]]" /etc/fstab | grep -v "sec=krb5:krb5i:krb5p" | awk '{print $2}'
```

Comment:
Just like in bash regex, use `?` instead of `*` to specify quantifier for `[4]`.